core/sched.c

```diff
@@ -44,6 +44,7 @@ volatile int sched_num_threads = 0;
 
 volatile unsigned int sched_context_switch_request;
 
+/* keep symbol names in sync with OpenOCD src/rtos/riot.c */
```

Comment:
This should be at least above `sched_num_threads`
https://github.com/RIOT-OS/RIOT/blob/3c533e1764b38adbd83505baaf37ef58ede63c9d/core/sched.c#L43

And I think it could be a bit clearer which variables are used here and which not. Maybe bundle them together and build a comment block around them

Reply:
Good idea, I grouped them together.